src/modelframe.jl

```diff
@@ -53,22 +53,26 @@ end
 _missing_omit(x::AbstractVector{T}) where T = copyto!(similar(x, nonmissingtype(T)), x)
```

Comment:
If I recall, the fact that this makes a copy even if `T` doesn't allow `Missing` was intentional to ensure that every column gets copied and downstream users of this function can mutate the result without worrying about affecting the original data. If we don't care about that, we can do
```suggestion
_missing_omit(x::AbstractVector{T}) where {T>:Missing} = copyto!(similar(x, nonmissingtype(T)), x)
_missing_omit(x::AbstractVector{T}) where {T} = x
```
without needing any other changes.